examples/sample_manifest_extended.json

```diff
@@ -0,0 +1,153 @@
+{
+  "metadata": {
+    "@type": "http://bib.schema.org/Audiobook",
+    "identifier": "https://librivox.org/flatland-a-romance-of-many-dimensions-by-edwin-abbott-abbott/",
+    "title": "Flatland: A Romance of Many Dimensions",
+    "author": {"name": "Edwin Abbott Abbott", "sort_as": "Abbott, Abbott Edwin"},
+    "narrator": "Abe Narrator; Beatrice Narrator",
+    "language": "en-US",
+    "description": "Flatland: A Romance of Many Dimensions is an 1884 science fiction novella by the English schoolmaster Edwin Abbott Abbott. As a satire, Flatland offered pointed observations on the social hierarchy of Victorian culture. However, the novella's more enduring contribution is its examination of dimensions; in a foreword to one of the many publications of the novella, noted science writer Isaac Asimov described Flatland as \"The best introduction one can find into the manner of perceiving dimensions.\" As such, the novella is still popular amongst mathematics, physics and computer science students. (Summary by Wikipedia)",
+    "subject": ["General Fiction", "Satire", "Science Fiction"],
+    
+    "typicalAgeRange": "7-11",
+    "publisher": "Librivox",
+    "published": "2008-10-12",
+    
+    "modified": "2016-08-11T19:32:18Z",
+
+    "belongs_to": {
+      "collection": "Math for All",
+      "series": {
+        "name": "Geometry Trilogy",
+        "position": 4
+      }
+    }    
+    
+    "duration": 15153,
+    "type": "audio/mpeg",
+    "bitrate": 128,
+    "samplerate": "22KHz",
+    "channels": 2,
+    
+    "contributor": [
+        {
+        "name": {
+            "ru": "Эдвин Эбботт Эбботт",
+            "en": "Edwin Abbott Abbott"
+        },        
+        "sort_as": {
+            "ru": "Эбботт, Эдвин Эбботт",
+            "en": "Abbott, Edwin Abbott"
+        },        
+        "scheme": "https://www.loc.gov/marc/relators/",
+        "role": ["aut"]
+        }, 
+        {
+        "name": "Edwin Abbott Abbott",
+        "scheme": "https://www.loc.gov/marc/relators/",
+        "role": ["led"]
+        },
+        {
+        "name": "Abe Narrator",
+        "scheme": "https://www.loc.gov/marc/relators/",
+        "role": ["nrt"]
+        },
+        {
+        "name": "Beatrice Narrator",
+        "scheme": "https://www.loc.gov/marc/relators/",
+        "role": ["nrt"]
+        }
+    ],
+
+    "schema:hasPart": [
+        {
+            "@type": "http://schema.org/VisualArtwork",
+            "schema:url": "cover.jpg",
+            "schema:license": "https://creativecommons.org/publicdomain/zero/1.0/"
+        },
+        {
+            "@type": "http://schema.org/Book",
```

Comment:
I think we'll need this.  Rights handling can and will be done in database on the server side, but it's a basic part of our functionality, and is bound to find its way into the manifest.  Better we provide a clean way now than scramble with additions later.